index.bs

```diff
@@ -268,33 +279,40 @@ xrDevice.requestSession({ immersive: true }).then((session) => {
 Session {#session}
 =======
 
-XRSessionCreationOptions {#xrsessioncreationoptions-interface}
+XRSessionMode {#xrsessionmode-enum}
 -------------------------
 
 <pre class="idl">
-dictionary XRSessionCreationOptions {
-  boolean immersive = false;
-  XRPresentationContext outputContext;
+enum XRSessionMode {
+  "inline",
+  "immersive-vr",
+  "immersive-ar"
 };
 </pre>
 
-The {{XRSessionCreationOptions}} dictionary provides a <dfn>session description</dfn>, indicating the desired properties of a session to be returned from {{requestSession()}}.
+The {{XRSessionMode}} enum defines the modes that an {{XRSession}} can operate in.
 
-<div class="algorithm" data-algorithm="supported-by-device">
+  - A session mode of <dfn enum-value for="XRSessionMode">inline</dfn> indicates that the session's output will be shown as an element in the HTML document. {{inline}} session content MAY be displayed in mono or stereo and MAY allow for [=viewer=] tracking. User agents MUST allow {{inline}} sessions to be created for any [=XR device=].
```

Comment:
Should it be a requirement that an outputContext be provided?

Reply:
Yes, at this point, but I'm hoping to strip that requirement out pretty quickly.